ui/ozone/platform/wayland/test/global_object.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef UI_OZONE_PLATFORM_WAYLAND_TEST_GLOBAL_OBJECT_H_
+#define UI_OZONE_PLATFORM_WAYLAND_TEST_GLOBAL_OBJECT_H_
+
+#include <memory>
+
+#include "base/macros.h"
+
+struct wl_client;
+struct wl_display;
+struct wl_global;
+struct wl_interface;
+struct wl_resource;
+
+namespace wl {
+
+struct GlobalObjectDeleter {
```

Comment:
unnamed namespace?

Reply:
This is the header; anonymous namespace would look weird here.

There is a better solution: I will move the deleter definition into the FakeServer class.